pandas/plotting/_misc.py

```diff
@@ -521,7 +521,7 @@ class _Options(dict):
     _ALIASES = {"x_compat": "xaxis.compat"}
     _DEFAULT_KEYS = ["xaxis.compat"]
 
-    def __init__(self, deprecated=False):
+    def __init__(self, deprecated=False) -> None:
```

Comment:
No problem, I think I can do it in separate PR as long as these are for fixing `__init__` return type annotation

Reply:
yeah, let's maybe keep the scope here limited to the return type of \_\_init__ since if there is consensus on #46337, there will be quite a bit to do (if not done in one big PR, https://github.com/pandas-dev/pandas/issues/46337#issuecomment-1065905362)